compose_api.go

```diff
@@ -122,10 +115,9 @@ func (r ComposeStackReaders) applyToComposeStack(o *composeStackOptions) {
 		if err != nil {
 			panic(err)
 		}
-		name := fmt.Sprintf("docker-compose-%d.yaml", idx)
-		filename := filepath.Join(tmpDir, name)
+		filename := filepath.Join(tmpDir, fmt.Sprintf("docker-compose-%d.yaml", idx))
 		if err := os.WriteFile(filename, content, os.ModePerm); err != nil {
-			continue
+			panic(err)
```

Comment:
Are we sure that we want to `panic` here? Probably there is a motivation that I'm not aware of, but I'd try to find a good reason to do so from the library standpoint not to surprise client code.

Reply:
It can be situation if file exists and access changed for it. Application has no permission write to file and I think no way ignore it. Or situation when no free space on disk.
Panic shows that library user should fix system problem before run tests.

Reply:
Thanks for the clarification :)